src/pages/page-2.js

```diff
@@ -3,13 +3,160 @@ import { Link } from "gatsby"
+            <input type="text" id="state" placeholder="State/Province*" />
+          </div>
+        </div>
+
+        <div className="flex flex-col md:flex-row">
+          <div className="divCell">
+            <input type="number" id="zip" placeholder="Postal / Zip code*" />
+          </div>
+          <div className="divCell">
+            <select defaultValue={"--Please Select--"}>
+              <option value="--Please Select--">--Please Select--</option>
+              <option value="North Dakota">North Dakota</option>
+              <option value="South Dakota">South Dakota</option>
+            </select>
+            <label className="text-gray-400">Country*</label>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div className="flex flex-col lg:flex-row lg:space-x-4">
+      {/* Products */}
+      <div className="flex flex-col lg:w-1/2 lg:flex-col">
+        <div>
+          <label className="sectionHeader">Products</label>
+          <hr className="sectionLine"></hr>
+        </div>
+        <div className="flex flex-col md:flex-row">
+          <div className="divCell">
+            <input type="text" id="item1name" placeholder="Item #1 name" />
+          </div>
+          <div className="divCell">
+            <input type="text" id="item1qty" placeholder="Item #1 quantity" />
+          </div>
+        </div>
+        <div className="flex flex-col md:flex-row">
+          <div className="divCell">
+            <input type="text" id="item2name" placeholder="Item #2 name" />
+          </div>
+          <div className="divCell">
+            <input type="text" id="item2qty" placeholder="Item #2 quantity" />
+          </div>
+        </div>
+        <div className="flex flex-col md:flex-row">
+          <div className="divCell">
+            <input type="text" id="item3name" placeholder="Item #3 name" />
+          </div>
+          <div className="divCell">
+            <input type="text" id="item3qty" placeholder="Item #3 quantity" />
+          </div>
+        </div>
+        <div className="flex flex-col md:flex-row">
+          <div className="divCell">
+            <label className="flex mb-2">Large Quote Upload</label>
+            <input type="file" />
+          </div>
+        </div>
+      </div>
+
+      {/* Notes */}
+      <div className="sectionSecondBlock flex flex-col lg:w-1/2 lg:flex-col">
+        <div>
+          <label className="sectionHeader">Notes</label>
+          <hr className="sectionLine"></hr>
+        </div>
+        <div className="divCell">
+          <label>Additional comments or questions</label>
+        </div>
+        <div className="divCell">
+          {/* <input type="textarea" rows="4" id="notes"/> */}
+          <textarea rows={6}></textarea>
+        </div>
+      </div>
+    </div>
+
+    <div>
+      <button className="bg-blue-500 uppercase text-xs rounded w-32 ml-2 mt-4">
+        Request Quote
+      </button>
+    </div>
+    <h1 className="py-24"></h1>
```

Comment:
not sure why a headline tag is here. Semantically, `<h1>` tags appear usually appear at the top of a page.

Reply:
I had used it to displaced (via the height) the code's generated footer away from my form, but I've now removed it.